### PR TITLE
Include initial ponyup package in lockfile

### LINKED ISSUE #88: ponyup show ponyup doesn't work after installation
After doing a fresh install of ponyup, `ponyup show ponyup` doesn't have any output. I had to run `ponyup update ponyup release` to install the 0.5.0 version before it would display any output. Note, that that is the same version that the installation process installs.

### PR DESCRIPTION
**Pull request by Theodus**
Closes #88

### CODE REVIEW

File: ponyup-init.sh [hunk 1]
```diff
@@ -95,7 +96,7 @@ ponyup_root="${prefix}/ponyup"
 echo "ponyup_root = ${ponyup_root}"
 
 mkdir -p "${ponyup_root}/bin"
-echo "${platform_triple}" > "${ponyup_root}/.platform"
```
Comment (SeanTAllen) [inline ponyup-init.sh]: any particular reason for this change?

File: ponyup-init.sh [hunk 2]
```diff
@@ -106,6 +107,17 @@ if [ "${response}" = "[]" ]; then
...
+ponyup_pkg="$(json_field "${response}" version)"
+ponyup_pkg="${ponyup_pkg}-${platform_triple_cpu}-${download_os##*-}"
+
+if [ "${repository}" = releases ]; then
+  ponyup_pkg="ponyup-release-${ponyup_pkg}"
+else
+  ponyup_pkg="ponyup-nightly-${ponyup_pkg}"
+fi
+echo "pkg: ${ponyup_pkg}"
+echo "${ponyup_pkg}" >"${ponyup_root}/.lock"
```
Comment (SeanTAllen) [inline ponyup-init.sh]: i personally always do a space before and after ">"

File: ponyup-init.sh [hunk 3]
```diff
@@ -106,6 +107,17 @@ if [ "${response}" = "[]" ]; then
...
 
+ponyup_pkg="$(json_field "${response}" version)"
+ponyup_pkg="${ponyup_pkg}-${platform_triple_cpu}-${download_os##*-}"
+
+if [ "${repository}" = releases ]; then
+  ponyup_pkg="ponyup-release-${ponyup_pkg}"
+else
+  ponyup_pkg="ponyup-nightly-${ponyup_pkg}"
+fi
+echo "pkg: ${ponyup_pkg}"
```
Comment (SeanTAllen) [inline ponyup-init.sh]: what's this echo? seems left over from debugging?

File: ponyup-init.sh (hunk 2 above)
Comment (Theodus) [reply ponyup-init.sh]: This was done automatically by my editor. will fix

File: ponyup-init.sh (hunk 3 above)
Comment (Theodus) [reply ponyup-init.sh]: correct. will fix

**Pull request merged**